### DISCUSSION TITLE
Questão TI21#7

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#13] [Q&A] [answered]
O que é o conceito de "closure" em JavaScript e por que é útil?

**Comment by Guilhlm** [comment]
Uma closure deixa você associar dados (do ambiente) com uma função que trabalha estes dados

**Comment by Lucasaraujo2** [comment, marked as answer]
Uma closure deixa você associar dados (do ambiente) com uma função que trabalha estes dados.
